libs/sr-ui/src/Hook/useLightCone.ts

```diff
@@ -0,0 +1,24 @@
+import { useContext, useEffect, useState } from 'react'
+import { DatabaseContext } from '../Context'
+
+export function useLightCone(lightConeId: string | '' | undefined = '') {
+  const { database } = useContext(DatabaseContext)
+  const [lightCone, setLightCone] = useState(
+    database.lightCones.get(lightConeId)
+  )
+  useEffect(
+    () => setLightCone(database.lightCones.get(lightConeId)),
+    [database, lightConeId]
+  )
+  useEffect(
+    () =>
+      lightConeId
+        ? database.lightCones.follow(
+            lightConeId,
+            (_k, r, v) => r === 'update' && setLightCone(v)
```

Comment:
I wonder if you should also listen to the "deletion" event as well, since that would also need to update UI?